examples/src/main/java/alluxio/examples/BasicOperations.java

```diff
@@ -83,7 +83,7 @@ private void writeFile(FileSystem fileSystem)
   private boolean readFile(FileSystem fileSystem)
       throws IOException, AlluxioException {
     boolean pass = true;
-    LOG.debug("Reading data...");
+    LOG.debug("Reading data...{}", mFilePath.toString());
```

Comment:
NIT: `mFilePath` please, first `LOG.debug` will call `toString` automatically; second, this is `LOG.debug`, and we want to introduce minimum overhead to non-debug code path. If we keep `mFilePath.toString()` here, no matter debug mode or not, this method is always called, which is unnecessary.